react-app/src/components/TextArea/index.js

```diff
@@ -27,24 +25,32 @@ export default class TextArea extends React.PureComponent {
     if (newText.length <= TextArea.MAX_LENGTH) this.setState({ currentText: newText });
   }
 
-  focus = () => this._textArea.focus();
```

Comment:
Why did you do this change?
This method (as well as `getText`) need to be bound to the instance in order to work. At least if they are called from a react controlled event or somewhere outside, basically anything other than `this.focus()` (or `this.getText()`) will fail.

Reply:
Same thing for the `getDate` method in the `DatePicker`.